clients/vault/src/oracle/constants.rs

```diff
@@ -36,20 +36,20 @@ pub fn get_min_externalized_messages(public_network: bool) -> usize {
 }
 
 /// --- Default values for the stellar overlay connection ---
-pub const OVERLAY_VERSION_PUBNET: u32 = 26;
-pub const MIN_OVERLAY_VERSION_PUBNET: u32 = 23;
+pub const OVERLAY_VERSION_PUBNET: u32 = 27;
+pub const MIN_OVERLAY_VERSION_PUBNET: u32 = 24;
 pub const LEDGER_VERSION_PUBNET: u32 = 19;
 pub const VERSION_STRING_PUBNET: &str =
-	"stellar-core 19.6.0 (b3a6bc28116e80bff7889c2f3bcd7c30dd1ac4d6)";
+	"stellar-core 19.7.0.rc1 (7249363c60e7ddf796187149f6a236f8ad244b2b)";
```

Comment:
The versions on PUBNET did not actually change yet. See eg [this](https://stellarbeat.io/nodes/GCGB2S2KGYARPVIA37HYZXVRM2YZUEXA6S33ZU5BUDC6THSB62LZSTYH?center=1&no-scroll=0&network=public) validator. So you'd have to revert these changes again.

Reply:
Thanks. Done.